### PR TITLE
Skywire a free and open internet.zh.md

### PR DESCRIPTION
**Pull request by Lezzkafka**
Chinese translate

### CODE REVIEW

**Comment by Lezzkafka** [comment]
@jayphbee Refined version, thanks for everything from the Skycoin chinese community！ You guys are the best ❤️

**Comment by Lezzkafka** [comment]
@weihuakevin 請查看哦 ：）

**Comment by weihuakevin** [comment]
approve

File: content/statement/Skywire A Free and Open Internet.zh.md [hunk 1]
```diff
@@ -13,88 +13,70 @@ categories = [
...
 
-A free internet has benefits that go beyond user experience. It provides a
-platform for innovation. Developers can create freely without being smothered
-by powerful corporations. When all sites are treated equally, the public
-benefits from increased transparency and information on topics ranging from
-policy to health. Creativity can flourish. Art becomes accessible to a broader
-demographic. Diverse perspectives can spread and be heard. Free speech can
-thrive. Users can choose their own experience and create their own content,
-rather than consuming a limited set of corporate products.
+免费的互联网有超越用户体验的好处。它提供了一个创新平台。开发人员可以自由地创造而不受压于大公司。
```
[the remark this answers is not in the document]
Comment (Lezzkafka) [reply content/statement/Skywire A Free and Open Internet.zh.md]: Thanks, i agree

**Comment by Lezzkafka** [comment]
SKY: 2SbFFB5mcCwt33BndK5ro7rbptEukkdKms9

**Pull request merged**